spec/support/factory_bot.rb

```diff
@@ -7,4 +7,13 @@
     password_confirmation { 'password' }
   end
 
+    factory :park do
+      sequence(:name) {|n| "#{n}" }
```

Comment:
Similarly, you don't actually have to string-interpolate a string if it's all alone :) I'd also suggest replacing these with something a bit more descriptive, otherwise your tests will easily fail -- if address and city are both "5", then which one are you verifying if you check that your page contains "5"?